pkg/sql/opt/exec/execbuilder/testdata/autocommit

```diff
@@ -37,7 +37,6 @@ WHERE message     LIKE '%r32: sending batch%'
   AND message NOT LIKE '%PushTxn%'
   AND message NOT LIKE '%QueryTxn%'
 ----
-dist sender send  r32: sending batch 1 CPut, 1 EndTxn to (n1,s1):1
```

Comment:
Hm - I don't quite follow why we're seeing differences to this test file?

Reply:
@RaduBerinde  Can you please take a look?

Reply:
Synching from our offline discussion - it looks like the query is specifically looking for messages from r32 which likely changed. We'll probably want to figure out which range these messages are on and update the query as well.